src/app/p-common/menubar/menubar.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { MenuItem } from 'primeng/api';
+
+@Component({
+  selector: 'app-menubar',
+  templateUrl: './menubar.component.html',
+  styleUrls: [ './menubar.component.css' ]
+})
+export class MenubarComponent implements OnInit {
+
+  constructor() {
+  }
+
+  items: MenuItem[];
```

Comment:
przypuszczam, że mityczny formatter doczepi się, że pole jest pod konstruktorem

Reply:
mój się nie czepia :P

Reply:
mój też nie, ale niektóre są bardziej czepialskie xd

Reply:
poza tym (moim zdaniem) ładniej to wygląda